pygments_routerlexers/Cisco/IOS.py

```diff
@@ -10,9 +10,9 @@ class IOSLexer(RegexLexer):
         "root" : [
             (r"^!.*", Comment),
             (r"^(version\s+)(.*)$", bygroups(Keyword, Number.Float)),
-            (r"(description)(.*?)$", bygroups(Keyword,Comment)),
+            (r"(desc*r*i*p*t*i*o*n*)(.*?)$", bygroups(Keyword,Comment)),
```

Comment:
If you just want description or desc you can do (desc(?:ription)?)(.*?)$  but i can see wanting to match on descr as well i guess?